### DISCUSSION TITLE
这个仓库是干嘛的？

### DISCUSSION THREAD

**Discussion by zth990219** [xcy960815/vscode-chatgpt-plugin#2] [Q&A] [answered]
这个仓库是干嘛的？

**Comment by xcy960815** [comment, marked as answer]
这个仓库是一个vscode编译器的插件，该插件的主要功能是将chatgpt 功能应用于vscode当中

**Reply by zth990219** [reply]
能否具体一点？

**Comment by xcy960815** [comment]
之前介于chatgpt 强大的能力，我也尝试过用 gpt 回答工作中的问题，发现超乎寻常的强大，所以我就在想将工作中常用的功能集成到vscode编译器当中，想我们在工作中常用的 查找bug、解释代码、优化代码、添加注释、补全代码等功能集成到vscode插件当中，当然了也预留了一些小入口支持自定义扩展

**Comment by zth990219** [comment]
知道啦，谢谢你的耐心解答
